includes/functions/metadata.php

```diff
@@ -394,31 +425,66 @@ function resource_data_init() {
 		]
 	);
 
+	// TODO: Validate year.
 	$cmb->add_field(
 		[
-			'name'        => __( 'Publication Date', 'learning-commons-framework' ),
-			'description' => __( 'The publication date of the resource in YYYY-MM-DD format.', 'learning-commons-framework' ),
-			'id'          => $prefix . 'publication_date',
-			'type'        => 'text_date',
-			'date_format' => 'Y-m-d',
+			'name'        => __( 'Publication Year (Required)', 'learning-commons-framework' ),
```

Comment:
Missing asterisk after "(Required)".

Reply:
@jhung This should be added by the pseudo-content from #110. Can you check again?

Reply:
Oops, found the problem. Bad merge. Fixed in https://github.com/platform-coop-toolkit/learning-commons-framework/pull/112/commits/f3c4a149db8ee8c14fd51d56c8a6589f098d83f4.